tools/mpremote/tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+# test outputs
+*.out
```

Comment:
Please don't add this file.  If the directory is dirty one can use `git clean -f` to get rid of them.

Reply:
ok, but at the risk of accidently adding .out files created during testing.